explore_simulation_approach/3_evaluate.py

```diff
@@ -33,6 +33,7 @@
 import pandas as pd
 import seaborn as sns
 import numpy as np
+import scipy
```

Comment:
Would this work?
```suggestion
import scipy.stats
```
Or `from scipy.stats import wilcoxon` would be fine too - it's just that`scipy` is a very large library so it probably isn't necessary to import the whole thing here, maybe better to stick with just the module or function you need.